test/.gitignore

```diff
@@ -0,0 +1,12 @@
+# Ignore everything
+*
+
+# ...even if they are in subdirectories
+!*/
```

Comment:
Actually the * would get subdirs as well, so we can just delete the line and the incorrect comment.